packages/core/security/core-security-common/src/roles/schema.ts

```diff
@@ -1,16 +1,16 @@
 /*
```

Comment:
Needed to move this file here to prevent cyclical dependencies. Is there a better approach? Or name to give to this file and it's parent?